.github/workflows/dapr-3rdparty-container.yaml

```diff
@@ -0,0 +1,41 @@
+name: dapr-mirror-containers
+
+on:
+  schedule:
+    - cron: '0 0 1 * *' # trigger on the 1st of every month at midnight
+  workflow_dispatch:
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Checkout
+        uses: actions/checkout@v2
+      - # ghcr logins for pushing image after testing
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Login to DockerHub
+        uses: docker/login-action@v1 
+        with:
+          username: ${{ secrets.DOCKER_REGISTRY_ID }}
```

Comment:
You can delete this login step. I assume we will only need the one for GHCR.

Reply:
I think we want this so the Docker pull step is authenticated and not getting the anonymous user rate limits.  You ok with that?

Reply:
Makes sense to me. Closing this comment.